crates/router/src/services/kafka/fraud_check_event.rs

```diff
@@ -0,0 +1,67 @@
+// use diesel_models::enums as storage_enums;
+use diesel_models::{
+    enums as storage_enums,
+    enums::{FraudCheckLastStep, FraudCheckStatus, FraudCheckType},
+    fraud_check::FraudCheck,
+};
+use time::OffsetDateTime;
+
+#[derive(serde::Serialize, Debug)]
+pub struct KafkaFraudCheckEvent<'a> {
```

Comment:
can use KafkaFraudCheckEvent everywhere and remove KafkaFraudCheck

Reply:
Actually i followed the same structure of event generation in kafka